cider/exec/template/GroupByAndAggregate.cpp

```diff
@@ -747,23 +748,6 @@ std::unique_ptr<QueryMemoryDescriptor> GroupByAndAggregate::initQueryMemoryDescr
                                        output_columnar_hint,
                                        /*streaming_top_n_hint=*/true,
                                        co);
```

Comment:
how to fallback?

Reply:
StreamingTopNOOM exception is not thrown